airbyte-webapp/src/packages/cloud/views/workspaces/WorkspaceSettingsView/WorkspaceSettingsView.tsx

```diff
@@ -77,22 +77,35 @@ export const WorkspaceSettingsView: React.FC = () => {
                     />
                   )}
                 </Field>
-                <Buttons>
+                <Label className={styles.formItem}>
```

Comment:
small nit: the `Advanced mode` label feels a little bit too close to the `Workspace name` input. If you look at the User Settings page, you can see that there is a small gap/margin between the `Email` input and the `Email notifications` label. Could we add a similar gap in between these two components?